packages/solarwinds-apm/src/exporters/metrics.ts

```diff
@@ -45,16 +47,31 @@ export class MetricExporter extends OTLPMetricExporter {
       }
     }
   }
-  override selectAggregationTemporality(
-    instrumentType: InstrumentType,
-  ): AggregationTemporality {
-    switch (instrumentType) {
-      case InstrumentType.HISTOGRAM: {
-        return AggregationTemporality.DELTA
-      }
-      default: {
-        return super.selectAggregationTemporality(instrumentType)
-      }
-    }
+
+  override selectAggregationTemporality(): AggregationTemporality {
+    return AggregationTemporality.DELTA
+  }
+}
+
+const DEFAULT_CARDINALITY_LIMIT = 200
```

Comment:
Is 200 a best-guess for SWO since the AO exporter MAX_TAGS is capped to 50.

Reply:
200 was just a guess to have something in place, I'm not opposed to lowering it to 50